crunchy.go

```diff
@@ -158,6 +161,15 @@ func (v *Validator) Check(password string) error {
 	if countUniqueChars(password) < v.options.MinDiff {
 		return ErrTooFewChars
 	}
+	idx, err := foundInHIBP(password)
+	if v.options.CheckHIBP {
+		if idx == -2 {
```

Comment:
I would recommend to always check for `err` being set (not `nil`), and in that case pass the error on. That also means we could slightly change the `foundInHIBP` function signature. See another related comment below.

Reply:
fixed